packages/flutter_tools/lib/src/commands/build_mojo.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../flx.dart';
+import '../globals.dart';
+import 'build.dart';
+
+const String _OPTION_SNAPSHOTTER = 'snapshotter-path';
+const String _OPTION_TARGET = 'target';
+const String _OPTION_PACKAGES = 'packages';
+const String _OPTION_OUTPUT = 'output-file';
+const String _OPTION_SNAPSHOT = 'snapshot';
+const String _OPTION_DEPFILE = 'depfile';
+const String _OPTION_WORKING = 'working-dir';
+const List<String> _OPTIONS = const <String>[
+  _OPTION_SNAPSHOTTER,
+  _OPTION_TARGET,
+  _OPTION_PACKAGES,
+  _OPTION_OUTPUT,
+  _OPTION_SNAPSHOT,
+  _OPTION_DEPFILE,
+  _OPTION_WORKING
+];
+
+class BuildMojoCommand extends BuildSubCommand {
+  BuildMojoCommand({bool verboseHelp: false}) {
+    _OPTIONS.forEach((String option) => argParser.addOption(option));
```

Comment:
We should just write up the calls to `addOption`.  We're going to need to add help text to them eventually anyway.